lerobot/common/policies/act/modeling_act.py

```diff
@@ -467,10 +469,9 @@ def forward(self, batch: dict[str, Tensor]) -> tuple[Tensor, tuple[Tensor, Tenso
         if self.use_images:
             all_cam_features = []
             all_cam_pos_embeds = []
-            images = batch["observation.images"]
 
-            for cam_index in range(images.shape[-4]):
-                cam_features = self.backbone(images[:, cam_index])["feature_map"]
+            for cam_index in range(batch["observation.images"].shape[-4]):
```

Comment:
FYI, not relevant to this PR, but I just noticed this while trying to track changes to batch. I think it's not necessary to pollute the namespace with `images = batch["observation.images"]`

Reply:
Unless there is a good reason that I am missing, I would prefer to never write anything in `batch` (e.g. `batch["observation.images"] = torch.stack(...)`)

Reply:
I'm not sure I understand this. Once it's been copied in the current scope, we should be able to do anything with it (apart from mutating the values in-place as they are still not copies).